File: problems/1033_moving_stones_until_consecutive/solution.py
class Solution:
    def numMovesStones(self, a: int, b: int, c: int) -> list[int]:
        s = sorted([a, b, c])
        if s[2] - s[0] == 2:
            return [0, 0]
        min_moves = 1 if s[1] - s[0] <= 2 or s[2] - s[1] <= 2 else 2
        max_moves = s[2] - s[0] - 2
        return [min_moves, max_moves]

File: problems/1033_moving_stones_until_consecutive/Solution.java
import java.util.*;

class Solution {
    public int[] numMovesStones(int a, int b, int c) {
        int[] s = {a, b, c};
        Arrays.sort(s);
        if (s[2] - s[0] == 2) return new int[]{0, 0};
        int minMoves = (s[1] - s[0] <= 2 || s[2] - s[1] <= 2) ? 1 : 2;
        int maxMoves = s[2] - s[0] - 2;
        return new int[]{minMoves, maxMoves};
    }
}
